.gitlab/e2e.yml

```diff
@@ -140,6 +140,14 @@ k8s-e2e-otlp-main:
   script:
     - inv -e new-e2e-tests.run --targets $TARGETS $CONFIGPARAMS
 
+new-e2e-containers-dev:
+  extends: .new_e2e_template
+  rules: !reference [.on_dev_branch_manual]
+  needs: []
+  variables:
+    TARGETS: ./containers
+    CONFIGPARAMS: "-c ddagent:fullImagePath=datadog/agent-dev:${CI_COMMIT_REF_SLUG}-py3 -c ddagent:clusterAgentFullImagePath=datadog/cluster-agent-dev:${CI_COMMIT_REF_SLUG}"
```

Comment:
Very good point !
In theory, this job should declare the “docker publish” jobs as dependencies.
I tested that few times ago, I didn’t manage to get it working properly and, in the end, @DataDog/agent-platform explained me that it was not possible to declare a manual job as a dependency.

I don’t know if GitLab has evolved since my last try but this explanation is still present in the comment of our GitLab files: https://github.com/DataDog/datadog-agent/blob/4c9ad68326ae254ee80a844878535c7863602907/.gitlab/e2e.yml#L21-L23